pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -97,14 +97,26 @@ public static String substr(String input, int beginIndex) {
    * @param endIndex index at which substring should be terminated
    * @return substring from beginIndex to endIndex
    */
-  @ScalarFunction
+  @ScalarFunction(names = {"substr", "substring"})
   public static String substr(String input, int beginIndex, int endIndex) {
     if (endIndex == -1) {
       return substr(input, beginIndex);
     }
     return StringUtils.substring(input, beginIndex, endIndex);
   }
 
+  /**
+   * Join two input string with seperator in between
+   * @param input1
+   * @param input2
+   * @param seperator
+   * @return The two input strings joined by the seperator
+   */
+  @ScalarFunction(names = "concat_ws")
+  public static String concatws(String seperator, String input1, String input2) {
```

Comment:
if there's aliasing, then the function name itself also needs to be included ( if we want to say support directly using `concatWs` )

Reply:
I don't want to support `contactws`, that's not standard (for the other ones it was backwards compatibility that I kept the old name)

Reply:
got it.